styles.css

```diff
@@ -1,3 +1,8 @@
+@font-face {
+    font-family: FreeSans;
+    src: url("../../../../lib/default.ttf");
```

Comment:
This relative path is pretty confusing. Can you pls at least add a comment why this is a valid past :-)